File: code-samples/python/catalog@stock.py
import requests

url = "https://vendor-api-staging.jumia.com/catalog/stock"

querystring = {"token":"string","size":"10","productSids":"string"}

payload = ""
headers = {"Authorization": "Bearer "}

response = requests.request("GET", url, data=payload, headers=headers, params=querystring)

print(response.text)

File: code-samples/java/catalog@stock.java
HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("https://vendor-api-staging.jumia.com/catalog/stock?token=string&size=10&productSids=string"))
    .header("Authorization", "Bearer ")
    .method("GET", HttpRequest.BodyPublishers.noBody())
    .build();
HttpResponse<String> response = HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
System.out.println(response.body());